package.json

```diff
@@ -24,14 +24,20 @@
         "@babel/preset-typescript": "^7.16.0",
         "@types/react": "^17.0.37",
         "@types/react-dom": "^17.0.11",
+        "@typescript-eslint/eslint-plugin": "^4.0.1",
+        "babel-eslint": "^10.1.0",
```

Comment:
npm WARN deprecated babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.

Thoughts?

Reply:
I changes this to @babel/eslint-parser